mtp_cashbook/apps/cashbook/tests/test_screenshots.py

```diff
@@ -0,0 +1,284 @@
+                        'owner_name': 'Melanie Smith',
+                    },
+                    {
+                        'id': 2,
+                        'prisoner_number': 'A3142AZ',
+                        'prisoner_name': 'Mark Smith',
+                        'amount': 2500,
+                        'sender_name': 'Campbell V',
+                        'received_at': received_at,
+                        'credited_at': None,
+                        'refunded_at': None,
+                        'resolution': 'pending',
+                        'anonymous': False,
+                    },
+                    {
+                        'id': 3,
+                        'prisoner_number': 'A2231YK',
+                        'prisoner_name': 'Stanley Ward',
+                        'amount': 4000,
+                        'sender_name': 'James R',
+                        'received_at': received_at,
+                        'credited_at': None,
+                        'refunded_at': None,
+                        'resolution': 'pending',
+                        'anonymous': False,
+                    },
+                    {
+                        'id': 4,
+                        'prisoner_number': 'A4940OG',
+                        'prisoner_name': 'Bradley Dale',
+                        'amount': 2500,
+                        'sender_name': 'Glen West',
+                        'received_at': received_at,
+                        'credited_at': None,
+                        'refunded_at': None,
+                        'resolution': 'pending',
+                        'anonymous': False,
+                        'owner_name': 'Melanie Smith',
+                    },
+                    {
+                        'id': 5,
+                        'prisoner_number': 'A7829IW',
+                        'prisoner_name': 'Douglas Jarvis',
+                        'amount': 1500,
+                        'sender_name': 'James R',
+                        'received_at': received_at,
+                        'credited_at': None,
+                        'refunded_at': None,
+                        'resolution': 'pending',
+                        'anonymous': False,
+                    },
+                    {
+                        'id': 6,
+                        'prisoner_number': 'A8140GN',
+                        'prisoner_name': 'Roge Hopkins',
+                        'amount': 500,
+                        'sender_name': None,
+                        'received_at': received_at,
+                        'credited_at': None,
+                        'refunded_at': None,
+                        'resolution': 'pending',
+                        'anonymous': True,
+                    },
+                    {
+                        'id': 7,
+                        'prisoner_number': 'A5729BC',
+                        'prisoner_name': 'Josephine Donald',
+                        'amount': 500,
```

Comment:
could you make the amounts add up to £230 for the control total to match NOMISh screenshot?